webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js

```diff
@@ -0,0 +1,111 @@
+import { getURIQuery } from 'foremanReact/common/helpers';
+import { getTableItemsAction } from 'foremanReact/components/common/table';
+import API from 'foremanReact/API';
+import { addToast } from 'foremanReact/redux/actions/toasts';
+
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  TASKS_CONTROLLER,
+  SELECT_ROWS,
+  UNSELECT_ALL_ROWS,
+  UNSELECT_ROWS,
+} from './TasksTableConstants';
+import { getApiPathname } from './TasksTableHelpers';
+
+export const getTableItems = () =>
+  getTableItemsAction(
+    TASKS_CONTROLLER,
+    getURIQuery(window.location.href),
+    getApiPathname()
+  );
+
+export const cancelTaskAction = id => async dispatch => {
+  await dispatch(cancelTask(id));
+  dispatch(getTableItems());
+};
+
+export const cancelTask = id => async dispatch => {
+  dispatch(
+    addToast({
+      type: 'info',
+      message: __('Trying to cancel the task'),
+    })
+  );
+  try {
+    await API.post(`/foreman_tasks/tasks/${id}/cancel`);
+    dispatch(
+      addToast({
+        type: 'success',
+        message: __('Task cancelled'),
```

Comment:
It would be really helpful here to know which task was successfully canceled (especially since this is for bulk and not just a single task.)  I know the API only provides `{statusText: "OK"}` but I think this would be worth making the change, either by using our available data on the frontend or making the API give us the task name in the response.

Reply:
Added id to the toasts, thanks!